test/mediawiki/page_test.rb

```diff
@@ -208,3 +228,91 @@
     end
   end
 end
+
+describe 'ReplaceableContent' do
+  describe 'no whitespace is found after the template' do
+    let(:client) do
+      client = MiniTest::Mock.new
+      client.expect(
+        :get_wikitext,
+        FakeResponse.new(WIKITEXT_NO_WHITESPACE_AFTER_TEMPLATE),
+        ['Some Wiki page']
+      )
+      client
+    end
+    let(:section) do
+      MediaWiki::Page::ReplaceableContent.new(
+        client: client,
+        title: 'Some Wiki page',
+        template: 'Politician scraper comparison'
+      )
+    end
```

Comment:
It would be good if we could eliminate this duplication of the `client` and `section` setup. I've just pushed https://github.com/everypolitician/mediawiki-page-replaceable_content/commit/4309b3296be77a811546740fa1bc8d9368c0c920 which shows one possible way of doing that.